Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1020,6 +1020,11 @@ class BrowserViewController: UIViewController,
     /// on the tab bar to open a new tab or by pressing the home page button on the tab bar. Inline is false when
     /// it's the zero search page, aka when the home page is shown by clicking the url bar from a loaded web page.
     func showEmbeddedHomepage(inline: Bool) {
+        guard let isPrivate = browserViewControllerState?.usePrivateHomepage, !isPrivate else {
```

Comment:
this is false due to the state being set only if the feature flag for the felt privacy work is enabled. To test the private homepage, set `isPrivate = true`